app/services/metrics.py

```diff
@@ -0,0 +1,76 @@
+import rasterio
+import matplotlib.pyplot as plt
+import io
+from matplotlib.colors import ListedColormap
+from rasterio.mask import mask
```

Comment:
I think it's better to keep one import per package as suggested in [this answer](https://stackoverflow.com/a/7374648)